directives/meta.py

```diff
@@ -42,10 +44,19 @@ class AplusMeta(Directive):
         'late-time',
     }
 
+    chapter_format = re.compile(
```

Comment:
Does this accept hyphens (`-`)? I checked and it does not. However, A+ frontend allows both hyphens and dots in its validator.

* https://github.com/apluslms/a-plus/blob/288696c3b62a8976f4ffb075aad34dd3cf09231d/lib/validators.py#L11
* https://github.com/apluslms/a-plus/blob/288696c3b62a8976f4ffb075aad34dd3cf09231d/course/models.py#L605-L611
* https://github.com/apluslms/a-plus/blob/288696c3b62a8976f4ffb075aad34dd3cf09231d/exercise/exercise_models.py#L196-L201

Reply:
Fixed in https://github.com/apluslms/a-plus-rst-tools/commit/02493c71ba43c15372f01aaa7ae55936a0dc4264